Execution:

python core.py 2> error.log
